opencga-app/app/scripts/azure/arm/mongodb/azuredeploy.json

```diff
@@ -0,0 +1,341 @@
+            }
+        },
+        "virtualNetworkName": {
+            "type": "string",
+            "metadata": {
+                "description": "Virtual network name"
+            }
+        }
+    },
+    "variables": {
+        "vmName": "mongo-vm",
+        "nsgName": "mongo-nsg",
+        "nicName": "mongo-nic",
+        "publicIPAddressName": "mongo-publicip",
+        "publicIPAddressType": "Static",
+        "dnsPrefix": "[concat(parameters('dnsLabelPrefix'), uniquestring(resourceGroup().id))]",
+        "sshKeyPath": "[concat('/home/',parameters('adminUsername'),'/.ssh/authorized_keys')]",
+        "copy": [
+            {
+                "name": "dnsNames",
+                "count": "[parameters('clusterSize')]",
+                "input": {
+                   "fqdn": "[concat(variables('dnsPrefix'),copyIndex('dnsNames'),'.',parameters('location'),'.cloudapp.azure.com')]"
+                }
+            }
+        ]
+    },
+    "resources": [
+        {
+            "name": "[variables('nsgName')]",
+            "type": "Microsoft.Network/networkSecurityGroups",
+            "apiVersion": "2018-08-01",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "securityRules": [
+                    {
+                        "name": "Port_27017",
+                        "properties": {
+                            "protocol": "*",
+                            "sourcePortRange": "*",
+                            "destinationPortRange": "27017",
+                            "sourceAddressPrefix": "*",
+                            "destinationAddressPrefix": "*",
+                            "access": "Allow",
+                            "priority": 1020,
+                            "direction": "Inbound",
+                            "sourcePortRanges": [],
+                            "destinationPortRanges": [],
+                            "sourceAddressPrefixes": [],
+                            "destinationAddressPrefixes": []
+                        }
+                    },
+                    {
+                        "name": "default-allow-ssh",
+                        "properties": {
+                            "protocol": "TCP",
+                            "sourcePortRange": "*",
+                            "destinationPortRange": "22",
+                            "sourceAddressPrefix": "*",
+                            "destinationAddressPrefix": "*",
+                            "access": "Allow",
+                            "priority": 1000,
+                            "direction": "Inbound",
+                            "sourcePortRanges": [],
+                            "destinationPortRanges": [],
+                            "sourceAddressPrefixes": [],
+                            "destinationAddressPrefixes": []
+                        }
+                    },
+                    {
+                        "name": "Port_80",
```

Comment:
Waht is port 80 needed for? Shame can't have comments!

Reply:
This port needs to be open so letsencrypt can issue the SSL certificate (and needs to remain open so this SSL certificate can be renewed)

Reply:
It requires inbound via port 80 - not just outbound?

Reply:
Removed NSGs for now in https://github.com/opencb/opencga/pull/975/commits/753dc108567e15b7f0e6d604ff3c2f99adda42c5